docs/docs/introduction/running.md

```diff
@@ -0,0 +1,22 @@
+---
+id: running
+title: Running Instant OpenHIE
```

Comment:
Should this not rather read `Running InstantHIE`?

Reply:
I don't think the name is really decided yet. I see we are using a mix of the two in the docs at the moment.

Reply:
Ok cool makes sense, we can create a ticket to update this through when a proper name is decided :+1: